src/Internal.AspNetCore.SiteExtension.Sdk/build/Internal.AspNetCore.SiteExtension.Sdk.targets

```diff
@@ -17,6 +17,7 @@
       <_TemplateFiles Include="$(MSBuildThisFileDirectory)..\content\HostingStartup\**\*" />
       <_HostingStartupPackageReference
         Include="%(HostingStartupPackageReference.Identity)"
+        Source="%(HostingStartupPackageReference.Source)"
```

Comment:
Would the source be different between each hosting startup?

Reply:
Not in our case, wanted to keep things flexible.